src/content/docs/en/installation/hardware.mdx

```diff
@@ -0,0 +1,117 @@
+
+- 4GB of RAM (8GB or more recommended)
+- 64-bit processor (x86_64/amd64)
+- 10GB of free disk space (20GB or more recommended)
+- Internet connection (for downloading the ISO image, updates, and additional drivers)
+- A computer that supports booting from a USB drive or DVD (most computers made after 2007 should support this feature)
+- A DVD or USB drive (for creating the bootable media)
+
+### Minimum Requirements for x86 Chromebooks
+
+- A [supported device](wiki.ultramarine-linux.org/chromebook/supported-devices)
+- USB Port **OR** SD Card Reader
+- 8gb of disk space (Total space, not what is usable in ChromeOS)
+- Ability to [enable developer mode](wiki.ultramarine-linux.org/chromebook/enabling-developer-mode)
+
+### Minimum Requirements for Raspberry Pi
+
+- Raspberry Pi 4 or newer
+- 4GB of RAM **OR** 2GB of RAM without a desktop environment
+- 8gb Class 10 MicroSD Card
+
+### Minimum Requirements for ARM Devices Using UEFI
+
+- 64-bit processor (aarch64)
+- 4GB or RAM
+- 10GB of Storage
+
+## Information on Nonstandard Hardware
+
+### Mac Computers with Apple Silicon
+
+Asahi Linux support on Ultramarine Linux is currently in internal-only alpha. Ultramarine Linux will likely use the upstream versions of Asahi packages in the near future.
+
+### ARM Chromebooks
+
+A closed alpha for the Kukui family of ARM Chromebooks will be opening in the near future.
+
+#### Devices in the Kukui Alpha Program
+
+- Kakadu (ASUS Chromebook Detachable CM3 (CM3000))
+- Krane (Lenovo Chromebook Duet/Lenovo IdeaPad Duet Chromebook First Generation)
+- Kodama (Lenovo 10e Chromebook Tablet)
+
+Contact [Owen Zimmerman](mailto:owen@fyralabs.com) for additional information
+
+### NVIDIA Graphics
+
+For computers with NVIDIA graphics cards, you have two choices for drivers.
+
+#### Noveau Driver
+
+[The Noveau driver](https://nouveau.freedesktop.org/) is a fully open source driver for NVIDIA graphics cards. It's built into Ultramarine so you can use your GPU instantly.
+
+Despite how good this sounds, modern NVIDIA GPUs will suffer performance issues and screen tearing under the Noveau driver.
+
+Older NVIDIA GPUs work well with the Noveau driver (and may only work with Noveau.)
+
+#### Proprietary Driver
+
+This is our recommended solution. Proprietary drivers support all of the features and performance of your GPU. They are automatically installed during installation if you are connected to the internet.
+
+If drivers weren't installed, or you switched to a NVIDIA GPU, you can install drivers by opening the "Software" app and searching for "NVIDIA" or by running the following commands in terminal:
+
+```bash
+sudo dnf update # Update the system first, the drivers may not work right if you don't.
+sudo dnf install akmod-nvidia # Install the NVIDIA kernel module.
+```
+
+<Alert type="warning">
+  You MUST wait until the process is finished before rebooting. Rebooting too
+  early might result in a broken installation! This may take upwards of 15
+  minutes!
```

Comment:
```suggestion
  minutes! To make sure the process is finished, open `htop` and look for the installation process.
```

Reply:
The user can also run `akmods` to check if the process is complete, maybe suggest that?

Reply:
Sure